src/main/java/com/yevhenberladyniuk/debts/service/PartnerService.java

```diff
@@ -1,21 +1,21 @@
 package com.yevhenberladyniuk.debts.service;
 
 import com.yevhenberladyniuk.debts.domain.Partner;
+import com.yevhenberladyniuk.debts.domain.User;
 import com.yevhenberladyniuk.debts.dto.CreatePartnerForm;
 import com.yevhenberladyniuk.debts.dto.PartnerDto;
 
 import java.util.List;
-import java.util.Optional;
 
 public interface PartnerService {
 
-    void create(CreatePartnerForm createPartnerForm, Long userId);
+    void create(CreatePartnerForm createPartnerForm, User user);
 
-    List<Partner> findAllByUserId(Long userId);
+    List<Partner> findAll(User user);
 
-    void deletePartnerById(Long id);
+    void deletePartnerById(Long id, User user);
 
-    Optional<Partner> findPartnerById(Long id);
+    Partner findPartnerById(Long id, User user);
 
-    void updatePartnerById(Long id, PartnerDto partnerDto);
+    void updatePartnerById(Long id, PartnerDto partnerDto, User user);
```

Comment:
You can skip the word 'Partner' in method signature, not necessary, but looks better